vllm/model_executor/models/gemma2.py

```diff
@@ -201,8 +201,8 @@ def __init__(
         self.mlp = Gemma2MLP(
             hidden_size=self.hidden_size,
             intermediate_size=config.intermediate_size,
-            hidden_act=config.hidden_act,
-            hidden_activation=config.hidden_activation,
+            hidden_act=getattr(config, "hidden_act", None),
+            hidden_activation=getattr(config, "hidden_activation", None),
```

Comment:
It looks like you'd need to change `Gemma2MLP` to take `Optional[str]` instead of strings and then change this check to allow None for either `hidden_act` or `hidden_activation` as well:

https://github.com/vllm-project/vllm/blob/4f5b059f146adeecd153fa781cf21863ed6679d8/vllm/model_executor/models/gemma2.py#L73

Reply:
Good catch, fixed